res/css/views/right_panel/_RoomSummaryCard.pcss

```diff
@@ -30,11 +30,12 @@ limitations under the License.
 
         .mx_RoomSummaryCard_roomName {
             margin: $spacing-12 0 $spacing-4;
-            font-weight: var(--font-semi-bold);
+            font-weight: var(--cpd-font-weight-semibold);
+            font-size: $font-17px;
         }
 
         .mx_RoomSummaryCard_alias {
-            font-size: $font-13px;
+            font: var(--cpd-font-body-md-regular);
```

Comment:
This sort of change confuses me, a font size is being added in one place and removed in another

Reply:
There's an issue with some of the tokens we have available to us. I want to discuss this further with @janogarcia when he's back from PTO